soccer/src/soccer/strategy/agent/agent_action_client.cpp

```diff
@@ -95,6 +99,10 @@ void AgentActionClient::world_state_callback(const rj_msgs::msg::WorldState::Sha
     last_world_state_ = std::move(world_state);
 }
 
+void AgentActionClient::play_state_callback(const rj_msgs::msg::PlayState::SharedPtr& msg) {
+    return;
```

Comment:
Hey amogh, can you also change the AC & Position to get play state data from this topic? the callback is empty right now; we should be forwarding it to position. I know position doesn't use play state data right now, but that's okay